10up/simple-podcasting
# feat/154: add a Podcast Platforms Block

Closes #154 



### Description of the Change



This PR adds the Podcast Platforms block.

#### Podcasting Platforms block

<img width="985" alt="Screenshot 2023-08-06 at 10 26 36 PM" src="https://github.com/10up/simple-podcasting/assets/17757960/df8a3795-010c-4b2f-ae86-5926146ba443">

#### Podcasting Platforms settings

<img width="551" alt="Screenshot 2023-08-06 at 10 24 43 PM" src="https://github.com/10up/simple-podcasting/assets/17757960/c2e097fa-56cc-4f9a-bf08-b27c0a4b1521">

### Verification Process



1. Add few podcasts.
2. On the podcasts edit screen, scroll down to the **Podcasting Platforms** section.
3. Set few platforms on which the podcast will be available.
4. Add the Podcast Platforms block to a page and select the podcast from the search field.
5. Confirm the output on the frontend, and also validate a11y.


### Checklist:






### Changelog Entry



### Credits


Props @jeffpaul @Sidsector9
